test/fakeintake/client/flare/flare.go

```diff
@@ -68,6 +69,16 @@ func (flare *Flare) FileHasContent(filename string) bool {
 	return flare.IsFile(filename) && flare.getFileInfo(filename).Size() > 0
 }
 
+// FileContains returns true if filename is a regular file and if its content contains a given pattern
+func (flare *Flare) FileContains(filename string, pattern string) bool {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Not related to this PR, we could revisit all test clients to use `t.Testing` and assert within calls, as `flare.AssertFileCOntains`

Reply:
I was thinking about this too (asked for inputs in `Additional Notes`). I'll follow up when creating the first flare tests